src/app.js

```diff
@@ -1 +1,26 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+const copyFile = (initialFile, targetFile) => {
+  if (initialFile === targetFile) {
+    console.error('Same location');
```

Comment:
This is a good check to ensure that the source and destination files are not the same. However, it might be beneficial to return or exit the function after logging the error to prevent further execution.